lib/BackgroundJob/BackgroundService.php

```diff
@@ -89,18 +89,33 @@ public function execute(int $timeout, bool $verbose, IUser $user = null, int $ma
 
 		// Here we are defining all the tasks that will get executed.
 		//
-		$task_classes = [
-			CheckRequirementsTask::class,
-			CheckCronTask::class,
-			LockTask::class,
-			DisabledUserRemovalTask::class,
-			StaleImagesRemovalTask::class,
-			CreateClustersTask::class,
-			AddMissingImagesTask::class,
-			EnumerateImagesMissingFacesTask::class,
-			ImageProcessingTask::class,
-			UnlockTask::class
-		];
```

Comment:
Introduce some var to make it more understandable ..

Maybe:

```
$singleTaskOrder = ($timeout > 0);
if ($singleTaskOrder) {
```

But the name doesn't convince me either .. Any other suggestions? :thinking:

Reply:
yes, maybe some comment why order is different. And you should probably document it.